app-frontend/src/app/components/common/search/search.html

```diff
@@ -2,14 +2,17 @@
   <i class="icon-search input-icon"></i>
   <input id="search-input"
          class="form-control"
-         ng-attr-placeholder="{{$ctrl.placeholder || 'Search'}}"
+         ng-attr-placeholder="{{$ctrl.searchText || $ctrl.placeholder || 'Search'}}"
```

Comment:
I don't see any value in adding searchText to the placeholder. If they delete the text, it will show their preview search, but I'd really expect it just to show the placeholder text and clear the search